packages/hardhat-plugin/src/ignition-wrapper.ts

```diff
@@ -1,26 +1,41 @@
 import {
-  Ignition,
+  Ignition as IgnitionCore,
   IgnitionDeployOptions,
   Providers,
   ExternalParamValue,
   Module,
   ModuleDict,
 } from "@ignored/ignition-core";
+import { Contract } from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { renderToCli } from "./ui/renderToCli";
 
 type HardhatEthers = HardhatRuntimeEnvironment["ethers"];
 
-export class IgnitionWrapper {
-  private _ignition: Ignition;
+interface Ignition {
+  deploy<T extends ModuleDict>(
+    ignitionModule: Module<T>,
+    deployParams?: {
+      parameters?: { [key: string]: ExternalParamValue };
+      ui?: boolean;
+    }
+  ): Promise<DeployResult>;
+}
```

Comment:
what's the purpose of this interface?

Reply:
Ah good point. I pulled it out so I could document it in the docs, but maybe it should now disappear again. I'll make that change.